src/pages/Acknowledgements/pageContent.tsx

```diff
@@ -147,12 +147,17 @@ export const pageContent = {
     content: (
       <>
         <Text size="medium" className="mt10">
-          <FormattedMessage defaultMessage="To become a validator you'll need to know about managing keys." />
+          <FormattedMessage
+            defaultMessage="To become a validator you'll need to know about managing keys and
+              protecting a mnemonic. If you are not yet familiar with keys and mnemomics, please
+              do not proceed."
+          />
         </Text>
         <Text size="medium" className="mt20">
           <FormattedMessage
-            defaultMessage="Your validator keys are derived from a uniquemnemonic (or seed). Your
-              seed will be the ONLY WAY to withdraw your funds. Above all, keep it safe!"
+            defaultMessage="Your validator keys are derived from a unique mnemonic (or seed). Your
```

Comment:
I wonder whether the paragraph below is needed in this section. 

```
"We'll help you create keys for each of your validators. These keys will be saved in password-encrypted keystore files and you will need to give these to your validator software to begin validating*. You will also receive a deposit file to upload to this website with the public keys for your validator."
```

Could it be something like: 

```
"We'll help you create keys for every validator you want to run. They keys are an important part of linking your ETH with your validator. They'll be generated via a mnemonic (seed) which you need to keep safe. Your mnemonic will be the only way to withdraw your ETH when the time comes and no one can help you recover your mnemonic if you lose it.
 ```
 
 This will condense the two paragraphs and remove redundant info (for now)